code/modules/ruins/objects_and_mobs/ruin_mapping_aids.dm

```diff
@@ -0,0 +1,14 @@
+//These landmarks can be placed in rooms/ruins to set the baseturfs of every turf in the area. Easier than having potentially unlimited subtypes of every turf or having to manually edit the turfs in the map editor
+
+/obj/effect/baseturf_helper
+	name = "lava baseturf editor"
+	icon = 'icons/obj/weapons.dmi'
+	icon_state = "syndballoon"
+	var/baseturf = /turf/open/floor/plating/lava/smooth/lava_land_surface
+
+/obj/effect/baseturf_helper/initialize()
+	..()
+	var/area/thearea = get_area(src)
+	for(var/turf/T in get_area_turfs(thearea, z))
```

Comment:
This proc not only loops through world, it also returns every turf (in world) under 'thearea' subtypes. Are you sure you want this?